src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div style="display: flex;">
+  <div>Octogons shapes go here</div>
+  <div>
+    <p>When you’re ready to apply we need two things:</p>
+    <ol>
```

Comment:
@will0101 Bit of invalid markup here. Only `li` elements can be direct children of `ol` elements.

Reply:
Fixed.